### ISSUE TITLE
Option to change the Station Icons.

### ISSUE THREAD

**Issue by mikhoul** [segler-alex/RadioDroid#533]
An option to be able to change the station icons by our own would be useful since lot of time the default icon used by those station are ugly...

![image](https://user-images.githubusercontent.com/5678673/59303737-d3d3c480-8c64-11e9-90d4-3ae11536c23d.png)

Regards :octocat:

**Comment by morckx** [comment]
> An option to be able to change the station icons by our own would be useful since lot of time the default icon used by those station are ugly...
> 
What I usually do is to simply add another instance of the station in http://www.radio-browser.info/ with a proper icon. It's not so nice, but otoh, then others benefit from this as well.

When the settings re-design is through, we'll also provide more options to transform the existing icons to uniform proportions (by stretching/compressing and/or cutting) – I guess.

**Comment by mikhoul** [comment]
>  It's not so nice

Not really nice, it pollute the DB with dupe of stations... 

I was reading the other issues in the repo here and the devs don't answer for months on many issues sadly, I was thinking it was an active project with an active team behind.

The more easy solution for the users would be to integrate an option to add our own URL/Radio stations directly in RadioDroid, this way we will not depend on the backend  API for everything.

I even opened an issue about this but I will not hold my breath... https://github.com/segler-alex/RadioDroid/issues/532

Regards :octocat:

**Comment by werman** [comment]
Well, project is not dead, things are not moving fast but they are moving. There are many issues that that should already be closed, some are indeed unanswered but they are monitored and considered.

**Comment by mikhoul** [comment]
[**@werman**](https://github.com/werman) commented on [12 juin 2019 à 13:44 UTC−4](https://github.com/segler-alex/RadioDroid/issues/533#issuecomment-501382366 "2019-06-12T17:44:40Z - Replied by Github Reply Comments"):
> Well, project is not dead, things are not moving fast but they are moving. There are many issues that that should already be closed, some are indeed unanswered but they are monitored and considered.

A simple answer to acknowledge that an issue has been read and considered is a minimum from my perspective.  How do you want that users/contributors feel involved if there is no "real" communication or ways to contribute ? 

Moving slow is one thing but leaving issues unacknowledged with no reply is another issue in itself.

**Comment by werman** [comment]
Yes, you are right. However knowing something and doing something are a two different things...
